lib/str-repr/decode.c

```diff
@@ -161,6 +162,7 @@ _process_backslash_escaped_character_in_strings(StrReprDecodeState *state)
 static gint
 _process_delimiter_characters_after_a_quoted_string(StrReprDecodeState *state)
 {
+  state->closing_delimiter = state->cur;
```

Comment:
does this point to the closing apostrophe or rather it is the delimiter e.g. the comma?